parsers/test/__snapshots__/test_ENTSOE.ambr

```diff
@@ -8373,7 +8373,7 @@
   list([
     dict({
       'correctedModes': list([
-        'gas',
+        <ProductionModes.GAS: 'gas'>,
```

Comment:
Do you think this will be a issue? if so I can make it so we call the `str()` method on them first to convert them to plain strings

Reply:
I would call string for backwards compatibility reasons, but I'm not sure if/where we use the corrected modes list.
The point about sets is good, but you probably have a better idea how often merging ProductionBreakdowns lists is. If it's not common, then I'd keep it as is

Reply:
It's pretty common actually as for example the ENTSOE parser creates one list per mode and then merges the mode specific lists into one final list.

It's very much dependent on the data structure of the parser.

I'll call str() on this to maintain backwards compatibility.

Reply:
Turns out the internal stuff already uses set for this so it's only the dict representation that uses a list, IDK why it does that but I don't think it's worth changing now.

Added a call to map to cast these to strings though.